src/main/java/step3/ResultView.java

```diff
@@ -0,0 +1,22 @@
+package step3;
+import java.util.List;
+
+public class ResultView {
+
+    public ResultView() {
+        System.out.println("실행결과");
+    }
+
+    public void printPosition(int positions){
+        for (int i = 0; i < positions; i++) {
+            System.out.println("-");
+        }
```

Comment:
```suggestion
        StringBuilder result = new StringBuilder();
        for (int i = 0; i < positions; i++) {
            result.append("-");
        }
        System.out.println(result.toString());
```
위와 같이 String Builder를 사용해보세요 !